pykechain/models/property.py

```diff
@@ -87,7 +92,30 @@ def value(self):
     @value.setter
     def value(self, value):
         # type: (Any) -> None
-        self._put_value(value)
+        if self.use_bulk_update:
+            self.__class__._update_package.append(dict(
+                id=self.id,
+                value=value,
+            ))
+            self._value = value
+        else:
+            self._put_value(value)
+
+    @classmethod
+    def update_values(cls, client: 'Client', use_bulk_update: bool = False) -> None:
```

Comment:
nice (but still would like to see it only on `Property2` as Property does not have this bulk update cha-thing-a-geh-shizzle anyway)